api-tests/data/cri-stub-requests/drivingLicence/kenneth-driving-permit-dva-valid/credentialSubject.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I accidentally addded this scenario - but I'm leaving it here as it may be useful in future.